doc/source/user/basics.types.rst

```diff
@@ -171,7 +118,7 @@ having unique characteristics.  Once you have imported NumPy using
 
 the dtypes are available as ``np.bool_``, ``np.float32``, etc.
 
-Advanced types, not listed in the table above, are explored in
+Advanced types, not listed in the site above, are explored in
```

Comment:
```suggestion
Advanced types, not listed above, are explored in
```

I think the use of the word "site" does not apply here, but I could be wrong :)

Reply:
Fixed.